src/styles/blocks/_stop-watch.scss

```
@@ -0,0 +1,100 @@
+@import "utils/variables";
+@import "utils/extends";
+
+.stop-watch {
+  position: absolute;
+  @include square($watch-size);
+  @extend %center;
+  background-color: ghostwhite;
+  border: 12px solid $secondary-color;
+  border-radius: 100%;
+  &__line {
+    background-color: $main-color;
+    position: absolute;
+    margin: auto;
+    &:first-child {
+      @extend %top;
+    }
+    &:nth-child(2) {
+      @extend %bottom;
+    }
+
+    &:nth-child(3) {
+      @extend %left;
+    }
+    &:nth-child(4) {
+      @extend %right;
+    }
+  }
+  &__center {
+    position: absolute;
+    @extend %center;
+    width: 40px;
+    height: 40px;
+    margin: auto;
+    background-color: $secondary-color;
+    border-radius: 100%;
+  }
+  &__hour {
```

put empty lines between classes
Sass has a minimization process & delete all spaces & comments into you file)

```suggestion
  border: 12px solid $secondary-color;
  border-radius: 100%;
  
  &__line {
    background-color: $main-color;
    position: absolute;
    margin: auto;
    
    &:first-child {
      @extend %top;
    }
    
    &:nth-child(2) {
      @extend %bottom;
    }

    &:nth-child(3) {
      @extend %left;
    }
    
    &:nth-child(4) {
      @extend %right;
    }
  }
  
  &__center {
    position: absolute;
    @extend %center;
    width: 40px;
    height: 40px;
    margin: auto;
    background-color: $secondary-color;
    border-radius: 100%;
  }
  
  &__hour {
```